.readthedocs.yml

```diff
@@ -20,7 +20,7 @@ formats: []
 python:
    version: 3.7
    install:
-      - requirements: requirements.txt
+      - requirements: docs/rtd-requirements.txt
```

Comment:
@leewujung looks like this is the suggested way to have a single req file for RTD. You still need to keep the docs/requirement.txt updated, but only for doc deps.

Reply:
Awesome, thanks! let me give it a try in a test RTD doc. Be back after a meeting...

Reply:
I updated the RTD min required version to 2.0, but haven't gotten the automodapi part to work : [build report](https://readthedocs.org/api/v2/build/10966609.txt). Was testing in [this branch](https://github.com/leewujung/echopype/tree/ocefpaf-fix_deps).

Reply:
The build report link opens a "not found" page for me :-/

Reply:
Sorry I guess that's probably somehow a private link in RTD: [new link](https://www.dropbox.com/s/9ajj1ag429qw6ov/10966609.txt?dl=0). 

After requesting to use sphinx>=2.0 (although RTD bulids using v1.8.3), it seems that automodapi is installed correctly. However there's an error for not finding content.rst, which is a little mysterious, because in `conf.py` I did set `master_doc = 'index'` and that should ask sphinx to use `index.rst`. 

P.S. The docs build locally.

Reply:
`docs/contents.rst` does nto exist on my fork :-/
I'm not sure how RTD works but this seems unrelated to this PR, no?

Reply:
What I meant was that there shouldn't be a need for a `docs/contents.rst`...
It is related to this PR, because the docs will fail to build if I merge it. 
Could you put sphinx-automodapi and numpydoc back to requirements.txt in the root level for now, and raise an issue on how they shouldn't be in there? I can make that issue too...

Reply:
Ah. You run `sphinx-automodapi` to create that content, right?

Reply:
? I think it autogenerates the context with the directive automodapi.